packages/core/test/model.test.ts

```diff
@@ -200,6 +200,31 @@ describe('createModel', () => {
     expect(updatedState.context.age).toEqual(42);
   });
 
+  it('can model actions', () => {
+    const userModel = createModel(
+      {
+        name: 'David',
+        age: 30
+      },
+      {
+        actions: {
+          greet: (message: string) => ({ message })
+        }
+      }
+    );
+
+    createMachine<typeof userModel>({
+      context: userModel.initialContext,
+      initial: 'active',
+      entry: { type: 'greet', message: 'hello' },
```

Comment:
this won't be executed here correctly, should we provide the model's implementations automatically to the machine? this would require supporting `model` top-level property as this would have to be accessible at runtime